src/config.ts

```diff
@@ -19,7 +19,7 @@ export interface Flags {
 
 export async function getConfig(flags: Flags) {
   // check to see if a config file path was passed
-  const configPath = flags.config || 'linkinator.config.json';
```

Comment:
Ok, now I'm convinced - whatever rule triggered this is bad.  In this case, if `flags.config` is an empty string, I *want* it to be replaced by the default.